### ISSUE TITLE
How to update Jellyfin in Truenas?

### ISSUE THREAD

**Issue by septarchy** [Thefrank/jellyfin-server-freebsd#14]
Hello Frank,

First of all: Thanks for your contributions and making these packages available for the community. really appreciate it! I've got jellyfin running for a couple months now and it works perfect. I do wonder what would be the best way to keep my installation up-to-date?

Would it be easy to create a guide or update script?

**Comment by Thefrank** [comment]
Glad everything is working out well for you!

The current version hosted here should be the latest stable version of jellyfin. I am typically able to update it within a week of any new version officially released. That being said, there really is no great way of keeping the FreeBSD version updated as there is a) no official FreeBSD version b) no auto-update feature and c) no version of it on ports/pkg.

Your best bet would be to download any newer version into the jail (e.g., via `fetch`), `pkg install jellyfin....` and finally `service restart jellyfinserver`

**Comment by grizewald** [comment]
> Your best bet would be to download any newer version into the jail (e.g., via `fetch`), `pkg install jellyfin....` and finally `service restart jellyfinserver`

I discovered a small problem with that idea when I just updated my installation. It resets all the previous setup, forcing you to add all the libraries again from scratch and forgetting all the watched/unwatched status, etc.

For the next time I need to update, is there any way to just update the package without making jellyfin server think it is a brand new installation?
